libs/blocks/global-footer/global-footer.js

```diff
@@ -90,6 +92,9 @@ class Footer {
     regionParent?.appendChild(region);
     socialParent?.appendChild(social);
 
+    const path = getFederatedUrl(url);
+    federatePictureSources({ section: this.body, forceFederate: path.includes('/federal/') });
```

Comment:
We have to separately call `decorateLinks` in the footer, so it's a bit of a special case. Any gnav element that is fetched (menus, nav, etc.) should usually always be decorated & have their picture sources federated